eth/api_miner.go

```diff
@@ -80,7 +64,7 @@ func (api *MinerAPI) SetEtherbase(etherbase common.Address) bool {
 	return true
 }
 
-// SetRecommitInterval updates the interval for miner sealing work recommitting.
-func (api *MinerAPI) SetRecommitInterval(interval int) {
-	api.e.Miner().SetRecommitInterval(time.Duration(interval) * time.Millisecond)
+// Etherbase is the address that mining rewards will be sent to.
+func (api *MinerAPI) Etherbase() common.Address {
+	return api.e.Miner().Etherbase()
 }
```

Comment:
I'd say this is not needed either, it's not the "real" etherbas,e it's the pending thing, so it's a bit misleading. We at best should retain a CLI flag, but IMO no need for API emthods.